### PR TITLE
Fix selector and serach bar

### PR DESCRIPTION
**Pull request by GusevPM**

### CODE REVIEW

File: src/views/contract/OperationsTab.vue [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <v-row>
       <v-col cols="9">
         <v-row v-if="searchable">
-          <v-col cols="12" class="pt-0">
```
Comment (aopoltorzhicky) [inline src/views/contract/OperationsTab.vue:7]: change `px-8 py-4` on `pa-8 ma-0` in `v-container` tag in the second line

File: src/components/ConnectWallet.vue [hunk 2]
```diff
@@ -6,7 +6,7 @@
         <v-icon size="26">mdi-wallet</v-icon>
       </v-btn>
       </template>
-      <v-list class="py-0" width="150">
```
Comment (aopoltorzhicky) [inline src/components/ConnectWallet.vue:9]: Remove width attribute

**Review by aopoltorzhicky — CHANGES_REQUESTED** [review]

File: src/views/contract/OperationsTab.vue (hunk 1 above)
Comment (GusevPM) [reply src/views/contract/OperationsTab.vue:7]: Done

File: src/components/ConnectWallet.vue (hunk 2 above)
Comment (GusevPM) [reply src/components/ConnectWallet.vue:9]: Done

**Review by aopoltorzhicky — APPROVED** [review]

**Pull request merged**